checador/app/helpers/companies_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CompaniesHelper
+end
```

Comment:
If you don't occupy this file, delete it.

Reply:
Thanks, the changes were made, could you check